eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -25,9 +25,9 @@ parameters:
     Linux_PyPy3:
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: 'pypy3'
-    Linux_Python38:
+    Linux_Python39:
       OSVmImage: 'ubuntu-18.04'
-      PythonVersion: '3.8'
+      PythonVersion: '3.9.0'
```

Comment:
Not sure it matters but is there a reason this is 3.9.0 instead of 3.9?

Reply:
Hm not sure. I copied this from the servicebus tests.yml. Will wait on final say from @scbedd

Reply:
IMO this is fine. I will check if `py39` has been added to the actual images themselves. We would use `3.9` like you're asking if it is already on the machine Wes.

Reply:
Given that this would need to be changed in 3 or so locations. This should probably just merge.